modules/schematics-core/utility/config.ts

```diff
@@ -145,3 +145,18 @@ export function getWorkspace(host: Tree): WorkspaceSchema {
 
   return JSON.parse(config);
 }
+
+export function setWorkspace(host: Tree, workSpace: WorkspaceSchema) {
+  const path = getWorkspacePath(host);
+  host.overwrite(path, JSON.stringify(workSpace, null, 2));
+}
+
+export function updateWorkspace(
```

Comment:
This is only used within the @ngrx/schematics package.
Wouldn't it be better to define these functions within the package itself?